Manual Testing/Ampeg Manual Test Project.txt

```diff
@@ -0,0 +1,68 @@
+Test Case #1 
+Create Account with Valid Information
+Entering valid information in create account page
+Pre conditions -Email, Username, First Name, Last Name, Password, Reenter Password, Country
```

Comment:
Not sure what these fields mean. Do I need these values before running the test?
A Pre-condition of this test is that the user account does not already exist.

Reply:
Got it, will change in the txt file before merge